src/java.base/share/classes/java/lang/classfile/instruction/InvokeInstruction.java

```diff
@@ -61,7 +64,7 @@ public sealed interface InvokeInstruction extends Instruction
     boolean isInterface();
 
     /**
-     * @return for an {@code invokeinterface}, the {@code count} value, as defined in {@jvms 6.5}
+     * {@return for an {@code invokeinterface}, the {@code count} value, as defined in {@jvms 6.5}}
```

Comment:
```suggestion
     * {@return the {@code count} value of an {@code invokeinterface} instruction, as defined in {@jvms 6.5}}
```

Reply:
What does this return for something that is not `invokeinterface` ? Do we infer the count from the descriptor? Or do we return -1 ? This should probably be reflected in the javadoc.